### DISCUSSION TITLE
Question: what section is used for changes to build requirements?

### DISCUSSION THREAD

**Discussion by hyperquantum** [olivierlacan/keep-a-changelog#508] [Q&A]
What if I want to mention any changes to the build requirements or dependencies of the project?
In what section do I put the following text:

- The project now requires at least Qt version 5.7

From _How do I make a good changelog?_ :
> `Changed` is for changes in existing functionality

Build dependencies are not about functionality, so I don't think a change like this really belongs in the `Changed` section.

So, does this actually require a new section? What should it be called? Or should information like this be omitted from the changelog?

**Comment by shadowspawn** [comment]
See also #234 

I think `Changed` is an appropriate section.

I think a change to the build requirements definitely belongs in the changelog as it affects the people who are using your project. A change just to the dependencies is less compelling, but consider whether some of the readers of the changelog will find it useful information. Remember, "Changelogs are for humans".

(I wondered about this the first time I had a dependency change in my own projects, and I decided `Changed` was an appropriate section.)
